services/ecosystems/config/default.json

```diff
@@ -0,0 +1,16 @@
+{
+  "db": {
+    "host": "localhost",
+    "port": 5432,
+    "user": "user",
+    "password": "password",
+    "database": "database"
+  },
+  "server": {
+    "port": 4004,
+    "origins": ["*"]
+  },
+  "logs": {
+    "dir": "logs"
+  }
+}
```

Comment:
You're missing the services object, please add them too in connectivity default config

Reply:
done